nvflare/apis/impl/job_def_manager.py

```diff
@@ -175,7 +179,25 @@ def _load_job_data_from_store(self, jid: str, temp_dir: str, fl_ctx: FLContext):
 
     def get_content(self, jid: str, fl_ctx: FLContext) -> bytes:
         store = self._get_job_store(fl_ctx)
-        return store.get_data(self.job_uri(jid))
+        meta = store.get_meta(self.job_uri(jid))
+        stored_data = store.get_data(self.job_uri(jid))
+        version = meta.get(JobMetaKey.VERSION, 1.0)
+        if version >= SimpleJobDefManager.JOB_VERSION:
+            return pickle.loads(stored_data).get(JobDataKey.JOB_DATA.value)
+        else:
+            return stored_data
```

Comment:
why do we need this version for?

Reply:
The version is to support the backward compatible purpose. The job data storage format will be changed with the workspace storage. Use the version info to support all the existing jobs and new jobs,

Reply:
I think we don't need the version number logic as no users are using the job concept today, I suggest remove these for clarity

Reply:
we can remove this later to avoid the unnecessary tickets, maybe in the 2nd build.

Reply:
Got it, thanks